assets/icon.svg

```diff
@@ -0,0 +1 @@
+<svg width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><title>Push Icon 01</title><g fill="none" fill-rule="evenodd"><path fill="#E63D2F" d="M0 0h128v128H0z"/><path fill="#C33B32" d="M0 104h128v24H0z"/><text font-family="Open Sans" font-size="16" font-weight="526" fill="#FFF"><tspan x="24.5" y="122">Web Push</tspan></text><g fill="#FFF"><g transform="translate(33 19)"><path d="M60.48 47.56c-.26-.032-4.292-.28-4.292-4.715v.278C53.502 14.43 39.184 7.14 31.067 7.14c-8.115 0-22.8 7.495-25.256 35.705 0 0-.877 4.624-4.29 4.714-.29.11-1.522.408-1.522 1.94v3.882s18.648-1.94 31-1.94 31 1.94 31 1.94V49.5c0-1.335-.818-1.85-1.52-1.94z"/><ellipse cx="31" cy="4.853" rx="4.428" ry="4.437"/></g><path d="M64 84.168c4.897 0 8.857-4.02 8.857-8.874 0 0-2.56-.97-8.857-.97-6.297 0-8.857.97-8.857.97 0 4.853 3.96 8.874 8.857 8.874z"/></g></g></svg>
```

Comment:
This should be in wp-web-push/lib, like bell.svg and bell_disabled.svg.

Can you also create an icon for the disabled state?